src/Parser/BloodDeathKnight/Modules/Core/VampiricBlood.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Module from 'Parser/Core/Module';
+import SPELLS from 'common/SPELLS';
+import ABILITYTRACKER from 'Parser/Core/Modules/AbilityTracker';
+import { calculateMaxCasts } from 'Parser/Core/getCastEfficiency';
+import ISSUE_IMPORTANCE from 'Parser/Core/ISSUE_IMPORTANCE';
+import SpellLink from 'common/SpellLink';
+
+const VAMPIRIC_BLOOD_COOLDOWN = 60;
+
+class VampiricBlood extends Module {
+  static dependencies = {
+    abilityTracker: ABILITYTRACKER,
+  };
+
+    suggestions(when) {
```

Comment:
I think this could also be solved by the CastEfficiency module, why is this separate?